src/test/java/com/wikia/webdriver/testcases/desktop/createawikitests/CreateWikiTestsLang.java

```diff
@@ -34,6 +34,7 @@ public void createNewWikiLangTC001(String lang) {
     cnw2.selectCategory(CreateWikiMessages.WIKI_CATEGORY_ID);
```

Comment:
This whole tests is 💩 . At least these name `CreateNewWikiPageObjectStepN` could be improved...